openmc/mesh.py

```diff
@@ -248,8 +248,10 @@ def write_data_to_vtk(self, points, filename, datasets, volume_normalization=Tru
 
         # create VTK arrays for each of
         # the data sets
+        datasets_out = []
```

Comment:
Can you add a comment above this explaining its purpose? Otherwise it will appear as though it serves no purpose.

Reply:
Good call! Just added that in.